bodhi/server/services/releases.py

```diff
@@ -29,16 +29,16 @@
     Package,
     Release,
 )
-import bodhi.server.schemas
-import bodhi.server.security
-import bodhi.server.services.errors
 from bodhi.server.validators import (
     validate_tags,
     validate_enums,
     validate_updates,
     validate_packages,
     validate_release,
 )
+import bodhi.server.schemas
+import bodhi.server.security
+import bodhi.server.services.errors
```

Comment:
Did you have a reason for moving these around?
I usually do the opposite, keep the ``import`` above the ``from ... import``, so I'm curious :)